src/locales/id.po

```diff
@@ -3136,83 +3107,101 @@ msgstr "Touch/No Touch"
+msgstr "bergerak keluar"
 
-msgid "Win payout if the last tick of %3 is strictly lower than the average of the %plural(%5,%d tick,%d ticks)."
-msgstr "Dapatkan hasil jika tik terakhir %3 pasti akan lebih rendah dari rata-rata %plural(%5,%d tick,%d ticks)."
+msgid "Win payout if %3 goes outside %7 to %6 through %5."
+msgstr "Dapatkan hasil jika %3 bergerak diluar %7 hingga %6 melewati %5."
 
-msgid "asian up"
-msgstr "asian up"
+msgid "Win payout if %3 goes outside %7 and %6 through %5 after %4."
+msgstr "Dapatkan hasil jika %3 bergerak diluar %7 dan %6 melewati %5 setelah %4."
 
-msgid "Win payout if the last tick of %3 is strictly higher than the average of the %plural(%5,%d tick,%d ticks)."
-msgstr "Dapatkan hasil jika tik terakhir %3 pasti akan lebih tinggi dari rata-rata %plural(%5,%d tick,%d ticks)."
+msgid "ends between"
+msgstr "berakhir antara"
+
+msgid "Win payout if %3 ends strictly between %7 to %6 at %5."
+msgstr "Dapatkan hasil jika %3 berakhir pasti antara %7 hingga %6 pada %5."
+
+msgid "Win payout if %3 ends strictly between %7 to %6 at %5 after %4."
+msgstr "Dapatkan hasil jika %3 pasti berakhir antara %7 hingga %6 pada %5 setelah %4."
+
+msgid "spread down"
+msgstr "spread turun"
+
+msgid "%1 %2 payout for every point %3 falls from entry level, with stop loss of %6 %4 and stop profit of %6 %5."
+msgstr "Hasil %1 %2 untuk setiap penurunan poin %3 dari entry level, dengan stop loss %6 %4 dan stop profit %6 %5."
+
+msgid "%1 %2 payout for every point %3 falls from entry level, with stop loss of %4 points and stop profit of %5 points."
+msgstr "Hasil %1 %2 untuk setiap penurunan poin %3 dari entry level, dengan stop loss %4 poin dan stop profit %6 %5."
+
+msgid "touches"
+msgstr "menyentuh"
+
+msgid "Win payout if %3 touches %6 through %5."
+msgstr "Dapatkan hasil jika %3 menyentuh %6 melewati %5."
+
+msgid "Win payout if %3 touches %6 through %5 after %4."
+msgstr "Dapatkan hasil jika %3 menyentuh %6 melewati %5 setelah %4."
 
 msgid "higher"
 msgstr "lebih tinggi"
 
-msgid "Win payout if %3 is strictly higher than %6 at %5."
-msgstr "Dapatkan hasil jika %3 pasti akan lebih tinggi dari %6 pada %5."
+msgid "Win payout if %3 is higher than or equal to %6 at %5."
+msgstr "Dapatkan hasil jika %3 lebih tinggi atau sama dengan %6 pada %5."
 
-msgid "Win payout if %3 is strictly higher than %6 at %5 after %4."
-msgstr "Dapatkan hasil jika %3 pasti akan lebih tinggi dari %6 pada %5 setelah %4."
+msgid "Win payout if %3 is higher than or equal to %6 at %5 after %4."
+msgstr "Dapatkan hasil jika %3 lebih tinggi atau sama dengan %6 pada %5 setelah %4."
 
-msgid "Win payout if %3 after %plural(%5,%d tick,%d ticks) is strictly higher than %6."
-msgstr "Dapatkan hasil jika %3 setelah %plural(%5,%d tick,%d ticks) pasti lebih tinggi dari %6."
+msgid "Win payout if %3 after %plural(%5,%d tick,%d ticks) is higher than or equal to %6."
+msgstr "Dapatkan hasil jika %3 setelah %plural(%5,%d tick,%d ticks) lebih tinggi dari atau sama dengan %6."
```

Comment:
Indonesian has only one form according to this header, no plural at all:

```
"Plural-Forms: nplurals=1; plural=0;\n"
```

So, there is no need to use the `%plural` function at all:

```
msgstr "Dapatkan hasil jika %3 setelah %5 tick lebih tinggi dari atau sama dengan %6."
```

or provide just one form:

```
msgstr "Dapatkan hasil jika %3 setelah %plural(%5,%d tick) lebih tinggi dari atau sama dengan %6."
```